test/index.test.js

```diff
@@ -1,8 +1,52 @@
-var createDMG = require('../');
-var assert = require('assert');
+const assert = require('assert');
+const electronDownload = require('electron-download');
```

Comment:
One option is to commit packaged app fixtures, like `electron-installer-debian`.